packages/nuxt/src/app/composables/ssr.ts

```diff
@@ -25,3 +25,10 @@ export function setResponseStatus (code: number, message?: string) {
     }
   }
 }
+
+export function setResponseHeader (name: string, value: string) {
```

Comment:
Not sure if we should add other types possibilities to the `value` argument, besides `string`.
For now this should be enough though.

Reply:
Could be maybe use [`setResponseHeader`](https://www.jsdocs.io/package/h3#setResponseHeader) right away?